docs/Catalog/SearchQuery.md

```diff
@@ -23,13 +23,13 @@ There are two types of indexing in Quilt:
 * *shallow* indexing includes object metadata (such as the file name and size)
 * *deep* indexing includes object contents. Quilt supports deep
 indexing for the following file extensions:
+  * .csv, .html, .json, .md, .rmd, .rst, .tab, .txt, .tsv (plain-text formats)
   * .fcs (FlowJo)
-  * .ipynb (Jupyter notebooks)
+  * .ipynb (Jupyter notebooks and Voila dashboards)
```

Comment:
Suggest reverting. Voila dashboards are not a file type and search the same as any Jupyter notebook.